Makefile

```diff
@@ -1,3 +1,10 @@
+include boilerplate/generated-includes.mk
+
+# format of full image is in ./boilerplate/openshift/golang-osd-operator/standard.mk
+#IMAGE_REPOSITORY=app-interface
+
+OPERATOR_NAME=route-monitor-operator
```

Comment:
The convention is to get this from the `config.go` file, e.g. https://github.com/openshift/aws-account-operator/blob/master/config/config.go

Reply:
I cannot take if from config, as it's a kustomize folder.. is there a place I can override where it takes it from?

Reply:
no, just override the name as you have done